spyder_terminal/widgets/main_widget.py

```diff
@@ -20,45 +21,50 @@
 from spyder.api.config.decorators import on_conf_change
 from spyder.api.widgets.main_widget import PluginMainWidget
 from spyder.config.base import get_translation, get_debug_level
+from spyder.utils.misc import select_port
 from spyder.utils.programs import find_program
 from spyder.widgets.tabs import Tabs
-from spyder.utils.misc import select_port
+from spyder.plugins.remoteclient.widgets import AuthenticationMethod
```

Comment:
I think we should move this enum to `spyder.plugins.remoteclient.api.protocol` to prevent breaking this or other external plugins in the future.

Do you prefer to do that for 6.1.2 or leave it for 6.2?

Reply:
Also the current minimum Spyder version that technically the plugin supports is 6.0.0. Due to that I ended up changing the use of `ClientType` for the `AuthenticationMethod` class here (https://github.com/spyder-ide/spyder-terminal/pull/358/commits/8c79b8843a32094b599630cabff96775a5c1cd9c). Following that, maybe should the Spyder minimum version requirement be changed in this PR too and instead use here the `ClientType` class?